api/src/main/java/org/openmrs/TestOrder.java

```diff
@@ -162,6 +162,8 @@ public TestOrder cloneForDiscontinuing() {
 		newOrder.setPreviousOrder(this);
 		newOrder.setPatient(getPatient());
 		newOrder.setOrderType(getOrderType());
+		newOrder.setOrderGroup(null);
```

Comment:
Clarification, no need to set null values, i would just not include these 2 lines